README.md

```diff
@@ -1,6 +1,19 @@
 HTTPS Everywhere [![Build Status](https://travis-ci.org/EFForg/https-everywhere.svg?branch=master)](https://travis-ci.org/EFForg/https-everywhere)
 ================
 
+Reproducing the Build
+---------------------
+
+Instead of modifying your own system, you may find it easiest to install docker to perform the building and testing steps outlined below from the same reproducible environment used by Travis CI.
+
+Based on the current `test/travis.sh`, you can build a docker image with `docker build -t httpse .` from the cloned `https-everywhere` directory.
+
+Then
+
+    docker run --interactive --tty --volume=$(pwd):/httpseverywhere --workdir=/httpseverywhere httpse
```

Comment:
You can also run the browser interactively using the steps here: http://stackoverflow.com/a/25280523

It would be worth it to mention that the several Firefox installations are located in the `/firefox-*` path.

So to run Firefox interactively from Docker, you can run:

    FIREFOX=/firefox-latest/firefox/firefox test/firefox.sh --justrun

To run Chromium within `Docker`, you need to add `--privileged` to the `docker run` command, since Chromium requires access to the kernel `lxc` APIs.